java/native-image/pom.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.4.0-M3</version>
```

Comment:
I'm guessing that since this is here, it works with M3.  The team didn't think it did, but good to see that this narrow example does 😄.

Reply:
Actually, downgrading to M2 removed the need for:
```
BP_BOOT_NATIVE_IMAGE_BUILD_ARGUMENTS="--initialize-at-build-time=org.springframework.boot.context.properties.bind.Bindable"
```
so I am going to go ahead and downgrade this example